### PR TITLE
Add worker CloudFormation stack

### PR DESCRIPTION
**Pull request by hectcastro**
The workers live within an Auto Scaling Group (ASG) that is currently using Amazon's base EC2 health check type. All workers live inside private subnets and generally only communicate with the outside world through messages passed via Redis.

Connects #296 and builds on top of #349, #350, #380, #383.

### CODE REVIEW

File: deployment/cfn/vpc.py [hunk 1]
```diff
@@ -47,11 +43,8 @@ class VPC(StackNode):
         'Region': 'us-east-1',
         'StackType': 'Staging',
         'KeyName': 'mmw-stg',
-        'IPAccess': '0.0.0.0',
+        'AvailabilityZones': ['us-east-1b', 'us-east-1d'],
```
Comment (kdeloach) [inline deployment/cfn/vpc.py]: This is a stupid question, but why default to availability zones b/d instead of a/b?

**Comment by kdeloach** [comment]
+1 Looks solid.

File: deployment/cfn/vpc.py (hunk 1 above)
Comment (hectcastro) [inline deployment/cfn/vpc.py]: Completely arbitrary.

**Comment by lliss** [comment]
+1

**Pull request merged**